justfile

```diff
@@ -1,93 +1,51 @@
-set positional-arguments
+set positional-arguments := true
+set dotenv-load := true
+
 alias t := test-all
 alias l := lint-all
 
-# Adding a chain
-add-chain:
```

Comment:
What is the complete user flow that replaces this command? I think the answer to that will help me review this pr overall. Is there any op-deployer in-flight work that needs to be merged to make it compatible with the SR 2.0? I noticed [this op-deployer pr](https://github.com/ethereum-optimism/optimism/pull/13519) was closed but not sure if there is a replacement